src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -115,7 +115,7 @@ public IEnumerable<User> Users
                     return new List<User>();
                 }
 
-                return UserTokenCache.GetUsers(ClientId);
+                return UserTokenCache.GetUsers(new Uri(Authority).Host);
```

Comment:
this is a change that is gonna affect all the platforms. I'm not seeing it documented anywhere that we're going to retrieve users with authority host instead of client id, first question is why we are doing it? Second is an ask, can you file an issue on all the platforms so we can make sure that all of us make the change.